src/redux/news/news.sagas.js

```diff
@@ -47,7 +47,7 @@ function* handleNewsLoad() {
   }
 }
 
-function* handleArticleLoad({ payload }) {
+export function* handleArticleLoad({ payload }) {
```

Comment:
Why have you added  `export` here? This saga function should be used only inside this file and root news saga

Reply:
I exported it for tests